cypress/e2e/ask_file/main.py

```diff
@@ -3,17 +3,22 @@
 
 @cl.on_chat_start
 def start():
-    file = None
+    txt_file = cl.AskFileMessage(
+        content="Please upload a text file to begin!", accept=["text/plain"]
+    ).send()
+    # Decode the file
+    text = txt_file.content.decode("utf-8")
+
+    cl.Message(
+        content=f"`Text file {txt_file.name}` uploaded, it contains {len(text)} characters!"
+    ).send()
 
-    # Wait for the user to upload a file
-    while file == None:
-        file = cl.AskFileMessage(
-            content="Please upload a text file to begin!", accept=["text/plain"]
-        ).send()
+    py_file = cl.AskFileMessage(
+        content="Please upload a python file to begin!", accept={"text/plain": [".py"]}
```

Comment:
Just a suggestion: you could add another test that submits a `text/plain` `.js` file to make sure it's not accepted.

Reply:
Added a test with a cpp file